scrapy/core/downloader/tls.py

```diff
@@ -48,6 +48,11 @@ def _identityVerifyingInfoCallback(self, connection, where, ret):
                         'Remote certificate is not valid for hostname "{}"; {}'.format(
                             self._hostnameASCII, e))
 
+                except ValueError as e:
+                    logger.warning(
+                        'SSL/TLS verification failed for hostname "{}"; {}'.format(
+                            self._hostnameASCII, e))
```

Comment:
I'm not sure it is Scrapy's job to implement verification - this code is better fit in Twisted or service_identity. I think we should figure out where this bug belongs, and document it as a bug, not as SSL/TLS verification failure.